homeassistant/components/logger/helpers.py

```diff
@@ -28,6 +28,9 @@
     STORAGE_VERSION,
 )
 
+SAVE_DELAY = 15.0
+SAVE_DELAY_LONG = 180.0
```

Comment:
Should we explain why 180 seconds is chosen?

Reply:
added a comment in 4c5b3c95e51340985a365a628006614a4e24b823